media-gfx/rawtherapee/rawtherapee-5.6.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils flag-o-matic toolchain-funcs xdg-utils
+
+MY_P=${P/_rc/-rc}
```

Comment:
lgtm, without having test-run myself yet. this line would be intuitively above `inherit` though, by convention of majorit of ebuilds.